docs/monitoring.md

```diff
@@ -79,12 +79,12 @@ If you're using *Kubernetes manifests* to install NGINX Kubernetes Gateway, you
 
 ## Available Metrics
 
-NGINX Kubernetes Gateway exports the following metrics:
+NGINX Gateway Fabric exports the following metrics:
 
 - NGINX metrics:
   - Exported by NGINX. Refer to the [NGINX Prometheus Exporter developer docs](https://github.com/nginxinc/nginx-prometheus-exporter#metrics-for-nginx-oss)
   - These metrics have the namespace `nginx_kubernetes_gateway`, and include the label `class` which is set to the
```

Comment:
We need to change the metrics namespace as well, https://github.com/nginxinc/nginx-kubernetes-gateway/blob/main/internal/mode/static/metrics/nginx.go#L27, and then these docs to match it.